### ISSUE TITLE
Module pulling in an old version of the SDK

### ISSUE THREAD

**Issue by heff1118** [aws/aws-sdk-java#1787]
Version 1.11.430 of the secert manager module is bringing in an older version of the core-java-sdk (1.11.125), I noticed this because there was a fix in 1.11.308 that was needed to get around this error: `Caused by: java.lang.NoSuchFieldError: SIGNING_REGION `

Here is my Pom setup:
``` 
<dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>com.amazonaws</groupId>
      <artifactId>aws-java-sdk-bom</artifactId>
      <version>1.11.430</version>
      <type>pom</type>
      <scope>import</scope>
    </dependency>
  </dependencies>
</dependencyManagement> 
```

and

```
<dependency>
     <groupId>com.amazonaws</groupId>
     <artifactId>aws-java-sdk-secretsmanager</artifactId>
</dependency>
```

When I run a mvn dependency tree I get a much older version of the core sdk:

```
[INFO] +- com.amazonaws:aws-java-sdk-secretsmanager:jar:1.11.430:compile
[INFO] |  +- com.amazonaws:aws-java-sdk-core:jar:1.11.125:compile
[INFO] |  |  +- commons-logging:commons-logging:jar:1.2:compile
[INFO] |  |  +- software.amazon.ion:ion-java:jar:1.0.2:compile
[INFO] |  |  \- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.8.11:compile
```

To resolve I explicitly included the version of the core-sdk that I needed:

```
<dependency>
	<groupId>com.amazonaws</groupId>
	<artifactId>aws-java-sdk-core</artifactId>
	<version>1.11.430</version>
</dependency>
```

**Comment by varunnvs92** [comment]
I just tested it with 1.11.430 and all the sdk dependencies have resolved to 1.11.430. Did you try with cleaning up the project?
Can you post the output of `mvn dependency:tree` command to debug further?

**Comment by debora-ito** [comment]
Closing this due to inactivity. Please feel free to reopen if you're still experiencing this issue.
